mysql-test/suite/tokudb/t/fast_update_binlog_mixed.test

```diff
@@ -5,13 +5,17 @@ source include/have_tokudb.inc;
 set default_storage_engine='tokudb';
 create table tt (id int primary key, x int);
 
-set session tokudb_disable_slow_upsert=1;
-
 connection master;
+source ../include/setup_fast_update_upsert.inc;
+set session tokudb_enable_fast_update=1;
```

Comment:
Perhaps this set session var set should go to that include file above

Reply:
No, because different tests require different sets of these variables.

Reply:
Then that file name might be misleading, because it does not "set up" fast updates and upserts: it just prepares for subsequent setup. But I don't have a suggestion for a better name.